chart/values.yaml

```diff
@@ -394,10 +394,11 @@ etcd:
   persistence:
     # -- If true, use a Persistent Volume Claim. If false, use emptyDir.
     enabled: true
-    # -- Will define which storageClass to use in etcd's StatefulSets
-    # a `manual` storageClass will provision a hostpath PV on the same node
-    # an empty storageClass will use the default StorageClass on the cluster
-    storageClass: ""
+    # -- Will define which storageClass to use in etcd's StatefulSets. Options:-
```

Comment:
Is the `-` and the end necessary?

Reply:
`-` can be removed. What do you mean by the end?

Reply:
I meant at the end of the line